db/schema.rb

```diff
@@ -23,4 +23,10 @@
     t.datetime "updated_at",      null: false
   end
 
+  create_table "users", force: :cascade do |t|
```

Comment:
I have no idea from where these table comes there are no migrations created and the 20160215122448_create_admin_users contains only the admin stuff hmm...